kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -169,4 +169,45 @@ class ExecuteStatement(
   }
 
   override protected def eventEnabled: Boolean = true
+
+  private def startTimeoutCheck(isTimedOut: AtomicBoolean, interval: Long): Unit = {
+    val checkTimeout = new Runnable {
+      override def run(): Unit = {
+        if (queryTimeout > 0) {
+          val execTime = System.currentTimeMillis() - startTime
```

Comment:
I think `queryTimeout` should be calculated from the submission time (ie `createTime`).

Reply:
Changed to use existed `addTimeoutMonitor` method, which calculated from from execution time.